ReleaseChecklist.md

```diff
@@ -3,24 +3,27 @@
 ### Requirements
  - [ ] Lauchpad (Ubuntu One) account
  - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
  - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
 
+### Blog Post
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) for each important bug.
+
 ### Documentation check
  - [ ] Run `make linkcheck` from within `docs/` and fix any broken links it finds. Ignore false positives caused by `href` anchors and dummy links not meant to work.
 
-### Blog Post
- - [ ] Create a post on https://github.com/ethereum/solidity-blog and explain some of the new features or concepts.
-
 ### Changelog
- - [ ] Sort the changelog entries alphabetically and correct any errors you notice.
- - [ ] Create a commit on a new branch that updates the ``Changelog`` to include a release date.
- - [ ] Run ``./scripts/tests.sh`` to update the bug list.
- - [ ] Create a pull request and wait for the tests, merge it.
+ - [ ] Sort the changelog entries alphabetically and correct any errors you notice. Commit it.
+ - [ ] Copy the changelog into the blog post.
+ - [ ] Update the changelog to include a release date.
+ - [ ] Run ``scripts/update_bugs_by_version.py`` to regenerate ``bugs_by_version.json`` from the changelog and ``bugs.json``.
+ - [ ] Commit changes, create a pull request and wait for the tests. Then merge it.
 
 ### Create the Release
  - [ ] Create Github release page: https://github.com/ethereum/solidity/releases/new
- - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.8.5`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
+ - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.8.5`)
+   (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release).
+   Include a warning that the release is still in progress and binaries may not yet be available.
```

Comment:
Maybe we should fix a wording, to keep this consistent?

Reply:
Wording of what specifically? The point as a whole is not like how I'd word it but it does not seem all that inconsistent to me.

But anyway, I cleaned this up a bit now - merged this with the previous checkbox, removed two parentheses that were next to each other and switched to double backticks. Please check if the new version is ok.

Reply:
Sorry, should have been clearer on that - I meant that we may want to fix the precise warning of the "release is still in progress" warning, s.t. it's always the same.

Reply:
Ah, ok. I added what I remember from the warning you used but I'm not sure if I didn't forget anything there. Please check.